packages/admin-ui/src/pages/wifi/components/wifi/WifiStatus.tsx

```diff
@@ -113,18 +108,18 @@ export default function WifiStatus(): JSX.Element {
         <Card spacing>
           <WifiInfo wifiStatus={wifiDnp.data.containers[0].state} />
           <hr />
-          <div className="wifi-status-actions-row">
-            <div className="wifi-status-container">
+          <div className="wifi-avahi-status-actions-row">
+            <div className="wifi-avahi-status-container">
               <StateBadgeContainer container={wifiDnp.data.containers[0]} />
-              <MdWifi className="wifi-status-icon" />
+              <MdWifi className="wifi-avahi-status-icon" />
```

Comment:
For consistency would be better to name everything as "local" instead of mixing local and avahi